app/controllers/oauths_controller.rb

```diff
@@ -28,9 +28,9 @@ def destroy
     authentication = current_user.authentications.find_by_provider(provider)
     if authentication.present?
       authentication.destroy
-      flash[:info] = "You have successfully unlinked your #{provider.titleize} account."
+      flash[:info] = t('auth.unlinked_successfully', provider: provider.titleize)
     else
-      flash[:alert] = "You do not currently have a linked #{provider.titleize} account."
+      flash[:alert] = t('auth.not_linked', provider: provider.titleize)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.